ArduPlane/tailsitter.cpp

```diff
@@ -39,10 +39,30 @@ bool QuadPlane::tailsitter_active(void)
  */
 void QuadPlane::tailsitter_output(void)
 {
-    if (tailsitter_active()) {
-        motors_output();
-        plane.pitchController.reset_I();
-        plane.rollController.reset_I();
+    if (!tailsitter_active()) {
+        return;
+    }
+
+    motors_output();
+    plane.pitchController.reset_I();
+    plane.rollController.reset_I();
+
+    if (tailsitter.input_mask_chan > 0 &&
+        tailsitter.input_mask > 0 &&
```

Comment:
yes, it was intentional. There are only 4 bits. If we end up with enough axes we can change this.